src/framework/components/model/component.js

```diff
@@ -223,12 +224,7 @@ pc.extend(pc, function () {
         },
 
         onSetLightmapped: function (name, oldValue, newValue) {
-            var model = this.data.model;
-            if (model) {
-                var meshInstances = model.meshInstances;
-                for (var i = 0; i < meshInstances.length; i++)
-                    meshInstances[i].lightmapped = newValue;
-            }
+            this._lightmapped = newValue;
```

Comment:
You shouldn't store data in the component, this should be stored in the data block.